components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -222,13 +235,14 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
                 >
                     <span
                         id={'channelHeaderDropdownIconLeft_' + id}
-                        className={'fa fa-angle-left SubMenu__icon-left' + (hasSubmenu && !isMobile && (direction === 'left') ? '' : '-empty' + (isMobile ? ' mobile' : ''))}
+                        className={classNames([`fa fa-angle-left SubMenu__icon-left${hasSubmenu && !isMobile && (direction === 'left') ? '' : '-empty'}`, {mobile: isMobile, 'sorting-menu-dropdown-icon': categorySortingMenu}])}
```

Comment:
Same comment here about classNames. Instead of a template string, you can just pass the `SubMenu__icon-left`/`SubMenu__icon-left--empty` as another argument

Reply:
in this case the class is part of the initial string so it wouldn't work passing it as an argument

Reply:
I tried but it doesn't work as an additional class, the CSS is only applied as an addition to the class, only one or the other can be set, we can make both conditional but IMO it will be more confusing?

Reply:
I don't know. I think a complicated ternary inside a template string inside a rather long line is pretty complicated as-is. Passing `(hasSubmenu && !isMobile && direction === 'left') ? 'SubMenu__icon-left' : 'SubMenu__icon-left-empty'` as an argument into classNames saves some of that nesting at least